README.md

```diff
@@ -46,6 +47,80 @@ npx truffle compile
 npm run networks-inject 
 ```
 
+# Deploying a simple market maker scenario to Rinkeby:
+
+The following script deploys a simple market maker order and a necessary owl order, to enable trading:
+```
+// Get token ID of DAI
+npx truffle exec scripts/stablex/invokeViewFunction.js 'tokenAddressToIdMap' '0x5592EC0cfb4dbc12D3aB100b257153436a1f0FEa' --network rinkeby
+
+// Export the resulting token ID
+export TOKEN_ID_DAI=[Result from last call]
+
+// Get token ID of TrueUSD
+npx truffle exec scripts/stablex/invokeViewFunction.js 'tokenAddressToIdMap' '0x0000000000085d4780B73119b644AE5ecd22b376' --network rinkeby
+
+// Export the resulting token ID
+export TOKEN_ID_TUSD=[Result from last call]
+
+// Make sure that the users have deposited sufficient funds into the exchange
+npx truffle exec scripts/stablex/deposit.js --accountId=0 --tokenId=0 --amount=30 --network rinkeby&& \
+npx truffle exec scripts/stablex/deposit.js --accountId=0 --tokenId=$TOKEN_ID_TUSD --amount=100 --network rinkeby
+
+// Place  market-maker order in current auction
+npx truffle exec scripts/stablex/place_order.js --accountId=0 --buyToken=$TOKEN_ID_DAI --sellToken=$TOKEN_ID_TUSD --minBuy=1000 --maxSell=998 --validFor=20 --network rinkeby
+
+// Place owl token order
+npx truffle exec scripts/stablex/place_order.js --accountId=0 --buyToken=$TOKEN_ID_DAI --sellToken=0 --minBuy=1000 --maxSell=1000 --validFor=20 --network rinkeby
+
+```
+
+Then, after switching to another account, a market order can be placed:
+```
+// Deposit funds into exchange:
+npx truffle exec scripts/stablex/deposit.js --accountId=0 --tokenId=$TOKEN_ID_DAI --amount=100 --network rinkeby
```

Comment:
accountID should be 1 here? Maybe also add that we expect this to happen through the UI?

Reply:
No, as the previous comment requires to export another PK. I don't expect people to work with only mnemonics